### PR TITLE
mathandwriter

### PR DESCRIPTION
**Pull request by Michaelnlearn**

### CODE REVIEW

File: Libreoffice/README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
-# LibreOffice
```
Comment (ArielHeleneto) [inline Libreoffice/README.md:1]: 按原来的

File: Libreoffice/README.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+
+LibreOffice 
 
 ## 软件说明
 
 LibreOffice（发音：/ˌliː.brəˈɒ.fɪs/）是由文档基金会开发的自由及开放源代码的办公室套件。LibreOffice套件包含文字处理器、电子表格、演示文稿程序、矢量图形编辑器和图表工具、数据库管理程序及创建和编辑数学公式的应用程序。
 
 ## 环境配置
 
-- 操作系统版本： openEuler 22.03 LTS riscv64
```
Comment (ArielHeleneto) [inline Libreoffice/README.md:9]: 按原来的格式啊。

File: Libreoffice/README.md [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
-- 软件版本： libreOffice 7.3.5.2 30(build:2)
+openEuler 22.03_LTS RISC-V
+5.10.0-7.oe2203.riscv64 (testing build 20220926)。
 
 ## 参考资料
 
-- [官网](https://www.libreoffice.org/)
```
Comment (ArielHeleneto) [inline Libreoffice/README.md:14]: 把官网删了干啥

**Review by ArielHeleneto — CHANGES_REQUESTED** [review]

**Review by ArielHeleneto — APPROVED** [review]
Accpeted

**Comment by Michaelnlearn** [comment]
thanks




------------------&nbsp;Original message&nbsp;------------------
From: "Ariel Xiong"; 
Sendtime: Wednesday, Nov 30, 2022 6:51 PM
To: "ArielHeleneto/RISCV-testcase"; 
Cc: ***@***.***>; "Author"; 
Subject: Re: [ArielHeleneto/RISCV-testcase] mathandwriter (PR #25)





 
@ArielHeleneto approved this pull request.
 
Accpeted

**Pull request merged**